Backend/src/main/java/DiceRoller.java

```diff
@@ -0,0 +1,48 @@
+package main.java;
+
+import java.util.Random;
+
+/**
+ * A local dice roller which generates a random number output based on the
+ * requested dice type
+ */
+
+public class DiceRoller {
+
+    private Random dice;
+
+    /**
+     * Initialize a DiceRoller object based on the random library
+     */
+    public DiceRoller() {
+        dice = new Random();
+    }
+
+    /**
+     * This function represents a dice roll. The theoretical range of the number
+     * of sided dice is infinite, so long as it is positive
+     * 
+     * @param diceCount The number of times we want a specific dice to be rolled
+     * @param modifier  The modifer that gets tagged at the end of a "dice roll"
+     * @param DiceType  The type of dice being rolled
+     * 
+     * @return Randomly generated number based on diceCount + the modifier + dice
+     *         type
+     */
+    int Roll(int diceCount, int modifier, enum DiceType) {
```

Comment:
Dice enum class, not enum 😀

Reply:
Yeah just realized it failed but my local code didn't say anything lol